modules/mobile/db/migrate/20210913194827_create_mobile_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMobileUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :mobile_users do |t|
+      t.string :user_id, null: false
+      t.datetime :created_at, null: false
```

Comment:
Could this use the built in Rails timestamps?
```suggestion
      t.timestamps null: false
```

Reply:
I used the created_at by itself since I didn't think updated_at would be relevant for us and timestamps creates both, but yeah I could definitely change it to use timestamps if we'd rather do that